styles/footer.scss

```diff
@@ -22,20 +28,29 @@
     }
   }
 
+  .footer-logo img {
+    margin: 0 auto;
+
+    @include breakpoint('md') {
+      margin: unset;
+    }
+  }
+
```

Comment:
add a fixed width on mobile for the logo
<img width="677" alt="Screen Shot 2023-03-21 at 10 18 49" src="https://user-images.githubusercontent.com/108662497/226577622-05a11aa6-2eca-4e59-97a1-cb6a28cc1e2b.png">